python/src/filesystem.rs

```diff
@@ -306,6 +307,19 @@ impl DeltaFileSystemHandler {
             .map_or(DEFAULT_MAX_BUFFER_SIZE, |v| {
                 v.parse::<usize>().unwrap_or(DEFAULT_MAX_BUFFER_SIZE)
             });
+        if max_buffer_size < DEFAULT_MAX_BUFFER_SIZE {
```

Comment:
It seems that below 5MB should never work, so shouldn't we just panic/raise here.

Reply:
It *could* work, a 5MiB (note: MiB) limit is mostly an implementation detail that most cloud providers share. It isn't necessarily set in stone by any spec or anything, you could self host your own minio and make the limit whatever you want. I'm just putting that warning there to make sure that end users know what they're getting into if they tweak around with things